cmd/govim/config/config.go

```diff
@@ -21,6 +21,11 @@ type Config struct {
 	// population. Default: false (0)
 	QuickfixAutoDiagnosticsDisable bool
 
+	// QuickfixSignsDisable is a boolean (0 or 1 in VimScript) that controls
+	// whether quickfix entries should be annotated with signs in the gutter.
+	// Default: false (0)
```

Comment:
Only one more small comment: I think we need to mention that this setting only has impact if the auto diagnostics are not disabled.

Reply:
I'll clarify the behaviour. It do actually have impact since it's possible to have signs enabled without the automatic diagnostics, i.e. placed when the user run `:GOVIMQuickfixDiagnostics`